plugins/source/gcp/client/spec.go

```diff
@@ -1,69 +1,7 @@
 package client
 
-import (
-	"time"
-
-	"github.com/googleapis/gax-go/v2"
-	"google.golang.org/api/option"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/backoff"
-)
-
 // Spec defines GCP source plugin Spec
 type Spec struct {
 	ProjectIDs            []string `json:"project_ids"`
 	ServiceAccountKeyJSON string   `json:"service_account_key_json"`
-
-	BaseDelay         int     `json:"backoff_base_delay,omitempty"`
```

Comment:
Oh, if we're removing these options, remember to also remove them from the config example here: https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gcp/resources/plugin/plugin.go#L22-L31

Reply:
good catch good catch